src/hooks/useCurrentPositionMarginRequirement.ts

```diff
@@ -1,17 +1,25 @@
 import { useState, useEffect } from 'react';
 import { AugmentedAMM } from '@utilities';
 
-export const useCurrentPositionMarginRequirement = (amm: AugmentedAMM) => {
+export const useCurrentPositionMarginRequirement = (
+  amm: AugmentedAMM,
+  fixedRateLower?: number,
+  fixedRateUpper?: number,
+) => {
   const [currentPositionMarginRequirement, setCurrentPositionMarginRequirement] =
     useState<number>();
 
   // Load the users balance of the required token
   useEffect(() => {
     const getData = async () => {
-      try {
-        const mrm = await amm.getPositionMarginRequirement(1, 999);
-        setCurrentPositionMarginRequirement(mrm);
-      } catch (e) {
+      if (fixedRateLower && fixedRateUpper) {
```

Comment:
is the check isUndefined missing here?

Reply:
doesn't it have the same effect?

Reply:
currently it also protects you against 0s, nulls, empty strings, not sure if this is desired

undefined && undefined === false
0 && 0 === false
null && null === false
'' && '' === false